src/scss/_mixins.scss

```diff
@@ -35,6 +35,14 @@
 		flex: 1 0 70%;
 		order: 2;
 	}
+}
+
+@mixin oTeaserSmallWithImage {
+	.o-teaser__heading:after {
```

Comment:
Can this be moved to the `oVideo` styles as well so they can be contained together